aspnetcore/blazor/tutorials/build-a-blazor-app.md

```diff
@@ -113,10 +109,8 @@ The `-n|--name` option in the preceding command specifies the name of the new Ra
 
 Open the `Todo` component in any file editor and make the following changes at the top of the file:
 
-<!-- UPDATE 8.0 For render mode guidance -->
-
 * Add an `@page` Razor directive with a relative URL of `/todo`.
-* Add the `[RenderModeServer]` attribute. The attribute indicates that this component should be rendered on the server with client interactivity.
+* Set the interactive Server render mode for the component. The render mode indicates that this component should be rendered on the server with client interactivity.
```

Comment:
```suggestion
* Set the interactive Server render mode for the component. The render mode indicates that this component should be rendered on the server with client interactivity.
```
```suggestion
* Enable interactivity on the page so that it isn't just statically rendered. The interactive server render mode enables the component to handle UI events from the server.
```